modules/administration/pages/monitoring.adoc

```diff
@@ -1,26 +1,20 @@
-// 2019-07-24, ke: Is it supported on Salt minions only?
-// This seems to indicate it: "Installing and configuring exporters is done using a Salt formula."
-// If yes, we'd better replace "mananged clients" with the term "Salt minions".
-
-[[prometheus]]
+[[monitoring]]
 = Monitoring with Prometheus
 
-Monitoring can be performed in {productname} using Prometheus and Grafana.
-The packages for Prometheus are shipped with SUSE Manager Client Tools, these packages also include several Prometheus exporters.
-{productname} Server and Proxy are now able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on managed client systems.
+You can monitor your {productname} environment using Prometheus and Grafana.
+{productname} Server and Proxy are able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on Salt clients.
 
-// I do not see why we should mention where this was started; we do not do this elsewhere
 Prometheus is a monitoring tool that is used to record real-time metrics in a time-series database.
-Unlike other monitoring systems, Prometheus collects metrics using HTTP pulls, allowing for higher performance and scalability.
-Prometheus is an open-source software project, mostly written in Go.
+Prometheus collects metrics using HTTP pulls, allowing for higher performance and scalability.
+Prometheus is an open-source software project, written in Go.
```

Comment:
Here we have three sentences all starting with Prometheus, but maybe it's not an issue.

Reply:
passive voice to the rescue! :tada: :wink: